client/src/features/Event/My/components/EventTabs.tsx

```diff
@@ -1,15 +1,25 @@
 import { css } from '@emotion/react';
+import { useQuery } from '@tanstack/react-query';
 
+import { myEventQueryOptions } from '@/api/queries/my';
 import { Tabs } from '@/shared/components/Tabs';
 
 import { UI_LABELS, STATUS_MESSAGES, TAB_VALUES } from '../constants';
-import { useEvents } from '../hooks/useEvents';
 
 import { EventSection } from './EventSection';
 import { EventTabsList } from './EventTabsList';
 
 export const EventTabs = () => {
-  const { events } = useEvents();
+  // E.TODO: organizationMemberId를 실제로 가져오는 로직 필요
+  const organizationMemberId = 1; // 임시 값
+
+  const { data: hostEvents = [] } = useQuery({
+    ...myEventQueryOptions.hostEvents(organizationMemberId),
+  });
```

Comment:
p2
객체/스프레드 연산자를 사용하지 않고, 아래처럼 바로 작성하는 것도 가능해 보이는데요! 혹시 객체로 감싸신 이유는 추후 확장성 등을 고려하신 걸까요??

```suggestion
  const { data: hostEvents = [] } = useQuery(
    myEventQueryOptions.hostEvents(organizationMemberId),
  );
```

Reply:
아뇨 그렇게 깊게 생각하지 않았고, useMutation 사용할 때의 습관이었습니다..
바로 제거하였습니다!

fb920eb0113363c1c918ef5cabd5b69285a6c6ce